observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -28,6 +28,45 @@ The following table provides information about which features are available depe
 |===
 
 
+[id="cluster-observability-operator-release-notes-0-4-1_{context}"]
+== {coo-full} 0.4.1
+
+The following advisory is available for {coo-full} 0.4.1:
+
+// Replace the errata link just before publishing
+* link:https://errata.devel.redhat.com/advisory/details/138912[RHEA-2024:???? {coo-full} 0.4.1]
+// * link:https://access.redhat.com/errata/RHEA-2024:????[RHEA-2024:???? {coo-full} 0.4.1]
+
+
+[id="cluster-observability-operator-0-4-1-new-features-enhancements_{context}"]
+=== New features and enhancements
+
+* You can now configure WebTLS for Prometheus and Alertmanager.
+
+[id="cluster-observability-operator-0-4-1-CVEs"]
+=== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2024-6104[CVE-2024-6104]
+* link:https://access.redhat.com/security/cve/CVE-2024-24786[CVE-2024-24786]
+
+[id="cluster-observability-operator-0-4-1-bug-fixes_{context}"]
+=== Bug fixes
+
+* Previously, when you tried to install COO UI plugins on {product-title} v4.16, the installation could fail. This release resolves the issue. (link:https://issues.redhat.com/browse/COO-451[*COO-451*])
```

Comment:
Previously we have no such issue, it's a issue introduced by fixing bug COO-152 in this release

Reply:
Removed